tests/Integration/Momento.Sdk.Tests/SimpleCacheControlTest.cs

```diff
@@ -9,24 +10,30 @@ namespace Momento.Sdk.Tests;
 public class SimpleCacheControlTest
 {
     private SimpleCacheClient client;
-    private string authToken;
+    private ICredentialProvider authProvider;
 
     public SimpleCacheControlTest(SimpleCacheClientFixture fixture)
     {
         client = fixture.Client;
-        authToken = fixture.AuthToken;
+        authProvider = fixture.AuthProvider;
     }
 
     [Fact]
     public void SimpleCacheClientConstructor_BadJWT_InvalidJwtException()
     {
-        Assert.Throws<InvalidArgumentException>(() => new SimpleCacheClient(Configurations.Laptop.Latest, "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJpbnRlZ3JhdGlvbiJ9.ZOgkTs", defaultTtlSeconds: 10));
+        Environment.SetEnvironmentVariable("BAD_MOMENTO_AUTH_TOKEN", "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJpbnRlZ3JhdGlvbiJ9.ZOgkTs");
+        Assert.Throws<InvalidArgumentException>(
+            () => new EnvironmentTokenProvider("BAD_MOMENTO_AUTH_TOKEN")
+        );
     }
 
     [Fact]
     public void SimpleCacheClientConstructor_NullJWT_InvalidJwtException()
     {
-        Assert.Throws<InvalidArgumentException>(() => new SimpleCacheClient(Configurations.Laptop.Latest, null!, defaultTtlSeconds: 10));
+        Environment.SetEnvironmentVariable("BAD_MOMENTO_AUTH_TOKEN", null);
+        Assert.Throws<InvalidArgumentException>(
+            () => new EnvironmentTokenProvider("BAD_MOMENTO_AUTH_TOKEN")
+        );
     }
```

Comment:
Is there a mechanism to setup and tear down the environment variables for the tests? My concern is when the test runner runs the tests concurrently.

The tests rely on shared state (environment variables), which are not isolated between the tests. If possible we should unset the variable after the test scope.

Reply:
It's already unset at the end of the second test, so we're good there. I added an explicit unset in the other test case in 37e2f6f.